Descent3/mission_download.cpp

```diff
@@ -843,11 +817,9 @@ int CheckGetD3M(char *d3m) {
 
```

Comment:
the `#if !defined(OEM)` check can be removed

Reply:
That will be outside of scope. OEM ifdefs should be treated on another PR.